easybuild/easyblocks/r/rust.py

```diff
@@ -51,6 +54,38 @@ def __init__(self, *args, **kwargs):
         self.cfg['build_cmd'] = "./x.py build"
         self.cfg['install_cmd'] = "./x.py install -j %(parallel)s"
 
+    def _convert_runpaths_to_rpaths(self):
+        """
+        Make sure that all shared libraries and excutable binaries use RPATH, not RUNPATH;
+        cfr. https://github.com/easybuilders/easybuild-easyconfigs/issues/18079
+        """
+        shlib_ext = get_shared_lib_ext()
+        shared_libs = glob.glob(os.path.join(self.installdir, 'lib', f'lib*.{shlib_ext}'))
+        executables = glob.glob(os.path.join(self.installdir, 'bin', '*'))
+        binary_files = [file for file in executables + shared_libs if is_binary(read_file(file, mode='rb'))]
```

Comment:
I would use `runpath_files` rather than `binary_files` (to avoid confusion of libraries with binaries vs binary files...)